src/marshal.rs

```diff
@@ -0,0 +1,68 @@
+use crate::conversion::AsPyPointer;
+use crate::ffi;
+use crate::{PyErr, PyObject, PyResult, Python};
+use std::os::raw::c_int;
+
+/// The current version of the marshal binary format.
+pub const VERSION: i32 = 4;
+
+/// Serialize an object to bytes using the Python built-in marshal module.
+///
+/// The built-in marshalling only supports a limited range of object.
+/// See the [python documentation](https://docs.python.org/3/library/marshal.html) for more details.
```

Comment:
I want a `#Example` section here.
It's difficult to guess how to specify the `version` without example.